### ISSUE TITLE
Accessibility issues on Cookiecutter Home Page: Search box and template list

### ISSUE THREAD

**Issue by eclectir** [Microsoft/PTVS#2223]
![image](https://cloud.githubusercontent.com/assets/5177491/23570244/38a594e6-0018-11e7-94ab-d5c77540851a.png)

**Comment by eclectir** [comment]
Error details-
BoundingRect - ControlIntersectsWithParent - The bounding rect of a control must not intersect with the bounding rect of its parent.
https://msdn.microsoft.com/en-us/library/system.windows.automation.automationelement.boundingrectangleproperty(v=vs.110).aspx

DuplicateAttributes - Sibling UIA_NameProperty - The sibling elements with the same type must not have duplicated element name property. Else the user will have difficulty to properly understand the control context.
https://msdn.microsoft.com/en-us/library/windows/desktop/ee684017(v=vs.85).aspx

MissingProperty - UIA_IsKeyboardFocusableProperty - Element cannot receive keyboard focus. The element might be reachable using the narrator but the user won't be able to invoke any action associated with the element using space or enter. On pressing the tab/arrow key,  the previous focused element will be used as the starting point. Even if the control can be invoked using other keyboard shortcuts, it is important for the control to be keyboard focusable because a user trying to invoke the control through a screen reader will accidentally invoke the previous control that has keyboard focus
https://msdn.microsoft.com/en-us/library/system.windows.automation.automationelement.automationelementinformation.iskeyboardfocusable(v=vs.110).aspx

**Comment by huguesv** [comment]
This has been fixed.
